client/state/reader/follows/actions.js

```diff
@@ -47,3 +52,52 @@ export function recordUnfollow( url ) {
 		} );
 	};
 }
+
+/**
+ * Returns an action object to signal that followed sites have been received.
+ *
+ * @param  {Array}  follows Follows received
+ * @return {Object} 		Action object
+ */
+export function receiveFollows( follows ) {
```

Comment:
question for my own enlightenment: 
why do we separate out `Receive` and `Successful`? Can't we also include the payload in Success?